### PR TITLE
[DOCS-1894] Docs for wandb beta leet terminal UI

### PR DESCRIPTION
**Pull request by mdlinville**
## Description
[DOCS-1894] Docs for `wandb beta leet` terminal UI

- CLI reference
- Keyboard shortcuts, including snippet
- Add instructions to narrative content about viewing runs, viewinvg system metrics
- Add abbreviated instructions to quickstarts
- Add tip to CLI overview
- Add screenshots

Graphics and details taken from https://github.com/wandb/wandb/pull/10764 and subsequent PRs

## Testing
- [x] Local build succeeds without errors (`mint dev`)
- [x] Local link check succeeds without errors (`mint broken-links`)
- [x] PR tests succeed

[DOCS-1894]: https://wandb.atlassian.net/browse/DOCS-1894?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: snippets/en/_includes/leet-keyboard-shortcuts.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+| **Key** | **Description** |
+| :--- | :--- |
+| `/` | Filter metrics by pattern (supports glob or substring matching) |
+| `ctrl+l` | Clear active filter |
+
+### Run Overview
+
+| **Key** | **Description** |
+| :--- | :--- |
+| `o` | Filter overview items (supports `@e`, `@c`, `@s` for environment/config/summary sections) |
```
Comment (dmitryduev) [inline snippets/en/_includes/leet-keyboard-shortcuts.mdx:36]:
```suggestion
| `o` | Filter overview items |
```

This is a bit half-baked (I mean, subsection filtering), I'm going to reworking it. Let's remove the reference for now.

File: snippets/en/_includes/leet-keyboard-shortcuts.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+| `R` | Set system grid rows (Shift+r) |
+
+### Run Overview Navigation (when sidebar open)
+
+| **Key** | **Description** |
+| :--- | :--- |
+| `up`, `down` | Navigate items in section |
+| `tab`, `shift+tab` | Switch between sections |
+| `left`, `right` | Navigate pages in section |
+| `@e`, `@c`, `@s` | Filter specific section (environment/config/summary) |
```
Comment (dmitryduev) [inline snippets/en/_includes/leet-keyboard-shortcuts.mdx:55]:
```suggestion
```

**Review by dmitryduev — APPROVED** [review]
Looks great, thanks @mdlinville!

File: models/ref/cli/wandb-beta.mdx [hunk 3]
```diff
@@ -21,5 +21,5 @@ Beta versions of wandb CLI commands. Requires wandb-core.
 
 | **Command** | **Description** |
 | :--- | :--- |
+| leet | Launch the W&B LEET (Lightweight Experiment Exploration Tool) terminal UI. |
```
Comment (ngrayluna) [inline models/ref/cli/wandb-beta.mdx]: Dumb question: Was this manually added because John's script to generate the CLI docs isn't out yet?

File: models/ref/cli/wandb-beta/wandb-beta-leet.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (ngrayluna) [inline models/ref/cli/wandb-beta/wandb-beta-leet.mdx:1]: Will this also get autogenerated with new CLI changes?

File: models/models_quickstart.mdx [hunk 5]
```diff
@@ -113,6 +113,12 @@ From here, you can see details about every run you've done. In this screenshot,
     <img src="/images/quickstart/quickstart_image.png" />
 </Frame>
 
+<Tip>
+Without leaving your terminal, you can view and monitor runs using the `wandb beta leet` terminal UI.
+![Screenshot demonstrating viewing a run in the LEET terminal UI](/images/leet/leet-tui.png)
+
+See the [`wandb beta leet`](/models/ref/cli/wandb-beta/wandb-beta-leet) reference for details.
```
Comment (ngrayluna) [inline models/models_quickstart.mdx]: Since it's in Beta, let's not prominently feature it on this page. We can consider adding it once it's full GA.

File: models/quickstart.mdx [hunk 6]
```diff
@@ -109,12 +109,27 @@ with wandb.init(project=project, config=config) as run:
         print(f"epoch={config['epochs']}, accuracy={acc}, loss={loss}")
         run.log({"accuracy": acc, "loss": loss})
 ```
-
-Visit [wandb.ai/home](https://wandb.ai/home) to view recorded metrics such as accuracy and loss and how they changed during each training step. The following image shows the loss and accuracy tracked from each run. Each run object appears in the **Runs** column with generated names.
+View recorded metrics such as accuracy and loss, and observe how they changed during each training step. You can view and monitor runs using the W&B App or the `wandb beta leet` terminal UI.
```
Comment (ngrayluna) [inline models/quickstart.mdx]: Same here. I don't think we should add this feature to the landing page since it's in Beta.

File: models/ref/cli/wandb-beta/wandb-beta-leet.mdx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+title: "wandb beta leet"
+description: "Use the W&B LEET (Lightweight Experiment Exploration Tool) terminal UI."
+---
+
+import LEETShortcuts from "/snippets/en/_includes/leet-keyboard-shortcuts.mdx";
+
+Launch the W&B LEET (Lightweight Experiment Exploration Tool) terminal UI.
+
+> _"htop, but for W&B runs."_
```
Comment (ngrayluna) [inline models/ref/cli/wandb-beta/wandb-beta-leet.mdx]: Sounds market-y

```suggestion
```

File: models/ref/cli/wandb-beta/wandb-beta-leet.mdx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+LEET is a fast local terminal UI for viewing a W&B run's stats, metrics, and system health. LEET is a keyboard-driven interface designed for users who work on remote systems or HPC environments using tools like SSH or `tmux`.
```
Comment (ngrayluna) [inline models/ref/cli/wandb-beta/wandb-beta-leet.mdx]:
```suggestion
LEET is a local terminal UI for viewing a W&B run's stats, metrics, and system health. LEET is a keyboard-driven interface designed for users who work on remote systems or HPC environments using tools like SSH or `tmux`.
```

File: models/ref/cli/wandb-beta.mdx (hunk 3 above)
Comment (ngrayluna) [reply models/ref/cli/wandb-beta.mdx]: For context, all content in cli/* is autogenerated from the source code. So any manual changes here will get overwritten by the source (if it's not there).

File: models/runs.mdx [hunk 9]
```diff
@@ -332,27 +332,72 @@ See [State fields](#run-states) for a full list of possible run states.
...
+
+1. Navigate to the directory containing your run, or note the path to the run directory or `.wandb` file.
+2. Run one of the following commands:
+
+   ```bash
+   # Look for the latest run in current directory
+   wandb beta leet
+   
+   # Specify a run directory
+   wandb beta leet ./wandb/run-20250813_124246-n67z9ude
```
Comment (ngrayluna) [inline models/runs.mdx:376]: Let's point out to users how to properly specify the run director (where and how to find it).

**Review by ngrayluna — CHANGES_REQUESTED** [review]

File: models/models_quickstart.mdx [hunk 10]
```diff
@@ -113,6 +113,12 @@ From here, you can see details about every run you've done. In this screenshot,
...
+</Tip>
```
Comment (mdlinville) [inline models/models_quickstart.mdx]:
```suggestion
```

File: models/quickstart.mdx [hunk 11]
```diff
@@ -109,12 +109,27 @@ with wandb.init(project=project, config=config) as run:
...
+<Tabs>
+<Tab title="App">
```
Comment (mdlinville) [inline models/quickstart.mdx]:
```suggestion
```

File: models/quickstart.mdx [hunk 12]
```diff
@@ -109,12 +109,27 @@ with wandb.init(project=project, config=config) as run:
...
+```bash
+wandb beta leet
+```
+
+![Screenshot demonstrating viewing a run in the LEET terminal UI](/images/leet/leet-tui.png)
+
+See the [`wandb beta leet`](/models/ref/cli/wandb-beta/wandb-beta-leet) reference for details.
+
+</Tab>
+</Tabs>
```
Comment (mdlinville) [inline models/quickstart.mdx]:
```suggestion
```

File: models/runs.mdx (hunk 9 above)
Comment (mdlinville) [reply models/runs.mdx:376]: Asked a followup about this in Slack. Thanks.

File: snippets/en/_includes/leet-keyboard-shortcuts.mdx (hunk 1 above)
Comment (mdlinville) [reply snippets/en/_includes/leet-keyboard-shortcuts.mdx:36]: Apologies, I needed to undo applying these suggestions manually to generated content. You will need to change this in the CLI source so we can pick it up next time we generate it.

File: models/runs.mdx (hunk 9 above)
Comment (mdlinville) [reply models/runs.mdx:376]: Done https://github.com/wandb/docs/pull/1893/files#diff-34ae33a0f602b6293c0c3f95f810904904e32f30ec99d76b0c6724fc6ce47836R366 (also in System Metrics)

File: models/ref/python/experiments/system-metrics.mdx [hunk 13]
```diff
@@ -12,6 +12,67 @@ This page provides detailed information about the system metrics that are tracke
...
+- RAM usage (GB)
+- Disk I/O
+- Network activity
+
+Get started with these keyboard shortcuts:
+- `]` - Toggle the right sidebar on/off
+- `alt+n` / `alt+pgdown` - Navigate to next system metrics page
+- `alt+N` / `alt+pgup` - Navigate to previous system metrics page
+- `C` - Configure the number of columns in the system metrics grid
+- `R` - Configure the number of rows in the system metrics grid
```
Comment (mdlinville) [inline models/ref/python/experiments/system-metrics.mdx]:
```suggestion
Get started with these keyboard shortcuts:
- `h` or `?` - View all keyboard shortcuts
- `/` - Filter metrics by pattern
- `[` / `]` - Toggle left/right sidebars
- `n` / `N` - Navigate between metric pages
- `q` / `CMD+C` - Quit
```

File: models/quickstart.mdx [hunk 14]
```diff
@@ -109,8 +109,7 @@ with wandb.init(project=project, config=config) as run:
         print(f"epoch={config['epochs']}, accuracy={acc}, loss={loss}")
         run.log({"accuracy": acc, "loss": loss})
 ```
-
-Visit [wandb.ai/home](https://wandb.ai/home) to view recorded metrics such as accuracy and loss and how they changed during each training step. The following image shows the loss and accuracy tracked from each run. Each run object appears in the **Runs** column with generated names.
+Visit [wandb.ai/home](https://wandb.ai/home). The following image shows the loss and accuracy tracked from each run. Each run object appears in the **Runs** column with generated names.
```
Comment (ngrayluna) [inline models/quickstart.mdx]: Looks like we are missing some text in the first sentence?

File: models/runs.mdx [hunk 15]
```diff
@@ -332,27 +332,74 @@ See [State fields](#run-states) for a full list of possible run states.
...
 ```
 
 Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and run ID.
 
+</Tab>
+<Tab title="LEET">
+
+To view a run locally in your terminal using the `wandb beta leet` terminal UI:
+
+1. If you started the run locally from a script, navigate to the directory where you ran your code. It contains a `wandb/` directory with a subdirectory per run and a `latest-run/` symbolic link. Each run directory contains a transaction log with the suffix `.wandb`.
```
Comment (ngrayluna) [inline models/runs.mdx]: Let's add that, in addition to the `.wandb` suffix, the run ID is prefixed to this file (`run-id.wandb`)

File: models/runs.mdx (hunk 9 above)
Comment (ngrayluna) [reply models/runs.mdx:376]: Thanks. Much better.

**Review by ngrayluna — COMMENTED** [review]
Looking much better. Minor suggestions.

File: models/quickstart.mdx (hunk 14 above)
Comment (mdlinville) [inline models/quickstart.mdx]:
```suggestion
Visit [wandb.ai/home](https://wandb.ai/home) to view recorded metrics such as accuracy and loss and how they changed during each training step. The following image shows the loss and accuracy tracked from each run. Each run object appears in the **Runs** column with generated names.
```

File: models/runs.mdx (hunk 15 above)
Comment (mdlinville) [inline models/runs.mdx]:
```suggestion
1. If you started the run locally from a script, navigate to the directory where you ran your code. It contains a `wandb/` directory with a subdirectory per run and a `latest-run/` symbolic link. Each run directory contains a transaction log named in the format `run-<run-ID>.wandb`.
```

File: models/ref/python/experiments/system-metrics.mdx [hunk 16]
```diff
@@ -12,6 +12,67 @@ This page provides detailed information about the system metrics that are tracke
...
+   - Network traffic
+
+You can customize which system metrics to display by adding panels to your workspace. See [Panels](/models/app/features/panels/) for more information on creating and customizing visualizations.
+
+</Tab>
+<Tab title="LEET">
+
+To view system metrics for a run in your terminal using the `wandb beta leet` terminal UI:
+
+1. If you started the run locally from a script, navigate to the directory where you ran your code. It contains a `wandb/` directory with a subdirectory per run and a `latest-run/` symbolic link. Each run directory contains a transaction log with the suffix `.wandb`.
```
Comment (mdlinville) [inline models/ref/python/experiments/system-metrics.mdx]:
```suggestion
1. If you started the run locally from a script, navigate to the directory where you ran your code. It contains a `wandb/` directory with a subdirectory per run and a `latest-run/` symbolic link. Each run directory contains a transaction log named in the format `run-<run-ID>.wandb`.
```

**Review by ngrayluna — APPROVED** [review]
Thanks!
